package.json

```diff
@@ -923,14 +924,17 @@
     "ts-jest": "^26.4.1",
     "ts-loader": "^6.2.2",
     "ts-node": "^8.6.2",
-    "typescript": "^4.0.3",
+    "typescript": "^4.1.3",
     "vsce": "^1.81.1",
     "vscode-test": "^1.3.0",
     "webpack": "^4.42.0",
     "webpack-cli": "^3.3.11",
     "xvfb-maybe": "^0.2.1"
   },
   "dependency-check": {
+    "entries": [
```

Comment:
ohh i see, maybe we need to bump depcheck inside the mongodb-js-precommit thing: newer versions probably supports ts: https://www.npmjs.com/package/depcheck

Reply:
Looks like that package might be a bit better than the one we're using: https://www.npmjs.com/package/dependency-check 
Would be good to be checking the `src` and not built `dist` files. I'll make a ticket for that: https://jira.mongodb.org/browse/COMPASS-4551

Reply:
Oh i see i thought we were using `depcheck`. But for VSCode i'd really consider just doing everything in project, i think having those simple support stuff in a separate package makes some sense in Compass cause of the 100 repos, but doesn't really offer much help here, especially considering it was build for a different tech stack.

Reply:
Agreed. I'll update to just use `depcheck` and we can remove the `mongodb-js-precommit` dependency :D
The eslint config is about the same with https://github.com/mongodb-js/mongosh/blob/master/config/eslintrc.base.js except for a rule or two. Might be a nice thing to do at some point, don't see much need right now.